code/modules/awaymissions/maploader/reader.dm

```diff
@@ -203,11 +244,10 @@ var/global/dmm_suite/preloader/_preloader = new
 	if(T)
 		//if others /turf are presents, simulates the underlays piling effect
 		index = first_turf_index + 1
-		while(index <= members.len)
-			turfs_underlays.Insert(1,image(T.icon,null,T.icon_state,T.layer,T.dir))//add the current turf image to the underlays list
-			var/turf/UT = instance_atom(members[index],members_attributes[index],xcrd,ycrd,zcrd)//instance new turf
-			add_underlying_turf(UT,T,turfs_underlays)//simulates the DMM piling effect
-			T = UT
+		while(index <= members.len - 1) // Last item is an /area
+			var/underlay = T.appearance
+			T = instance_atom(members[index],members_attributes[index],xcrd,ycrd,zcrd)//instance new turf
+			T.underlays += underlay
```

Comment:
Are you sure this will also handle the density/opacity piling effect (e.g if you put a dense turf over some base non-dense turf, or the other way around, the final turf is dense) ?